app/sign.js

```diff
@@ -103,7 +109,12 @@ const handleSignUtxo = function(recoveryRequest, key, skipConfirm) {
   // force override network as we use btc mainnet xpubs for all utxo coins
   backupKeyNode.keyPair.network = network;
 
+  transaction.ins.forEach(function (input, i) {
```

Comment:
What are we doing here? I don't see this attribute, `value`, appear anywhere else in the file. -- is this necessary? If so, add an inline comment as to why.